examples/04_tile_iterator/tile_iterator.cu

```diff
@@ -94,19 +94,7 @@ __global__ void copy(
 
     typename Iterator::Fragment fragment;
 
-    for(size_t i = 0; i < fragment.size(); ++i) {
-      fragment[i] = 0;
-    }
-
-    src_iterator.load(fragment);
-    dst_iterator.store(fragment);
-
-
-    ++src_iterator;
-    ++dst_iterator;
-
-    for(; iterations > 1; --iterations) {
-
+    for(; iterations > 0; --iterations) {
```

Comment:
we still need to keep the initialization part of code.  we want `fragment` is in a known state before the load and store operation.